package.json

```diff
@@ -2,21 +2,24 @@
   "name": "package.land",
   "private": true,
   "description": "A nice place for JavaScript packages",
-  "main": "index.js",
+  "main": "server.js",
   "repository": "https://github.com/nice-registry/package.land",
   "author": "zeke",
   "license": "MIT",
   "scripts": {
     "start": "node server.js",
-    "test": "mocha && standard --fix && standard-markdown"
+    "test": "NODE_ENV=test mocha && standard --fix && standard-markdown",
+    "dev": "nodemon"
   },
   "dependencies": {
     "all-the-package-repos": "^1.2.1",
-    "express": "^4.16.2"
+    "express": "^4.16.2",
+    "lil-env-thing": "^1.0.0"
```

Comment:
short just question, why not ~ instead of ^ by default? 

related . https://github.com/rvagg/npm-explicit-deps#wut